evals/index.eval.ts

```diff
@@ -776,12 +774,8 @@ const google_jobs: EvalFunction = async ({ modelName, logger }) => {
 
     const jobDetails = await stagehand.extract({
       instruction:
-        "Extract the following details from the job posting: application deadline, minimum qualifications (degree and years of experience), and preferred qualifications (degree and years of experience)",
```

Comment:
do we need to remove this? might be better to have a score that shows completion of each field?

Reply:
from huddle: not on page anymore